src/codec/RowWriterV2.cpp

```diff
@@ -16,6 +16,50 @@ namespace nebula {
 
 using nebula::cpp2::PropertyType;
 
+// Template functions implement support for list and set data types
+template <typename Container>
+WriteResult writeContainer(const Container& container, Value::Type valueType, std::string& buffer) {
+  std::unordered_set<Value> serialized;
```

Comment:
If the container is a List, we don't need the`unordered_set`. Can we be sure that the compiler can ignore it without affecting performance?

Reply:
OK, I have modified the implementation of two functions corresponding to List and Set respectively. Unordered_set is only used when necessary: ​​writeList() does not use unordered_set, writeSet() uses it to ensure data uniqueness